__tests__/transformdata.test.ts

```diff
@@ -0,0 +1,25 @@
+import { describe, expect, it } from '@jest/globals';
+import { transformJsonData } from '@/transform';
+
+describe('transformJson', () => {
+    it('should transform an array of objects into Json format', () => {
+        expect.hasAssertions();
+        const inputData = [
```

Comment:
Maybe lets move this to a separate file, i.e. `__tests__/data/dataFormats.ts` and call the `const` something like `jsonStandard`?